ui/job-view/context/Pushes.jsx

```diff
@@ -395,9 +397,12 @@ export class PushesClass extends React.Component {
 
     window.dispatchEvent(
       new CustomEvent(thEvents.applyNewJobs, {
-        detail: { jobs, updatedSelectedJob },
+        detail: { jobs },
```

Comment:
This is because the Redux store doesn't listen for these events like the Context did.  So we need a separate call for updating.